test/error-messages/invalid-arg-value.txt

```diff
@@ -0,0 +1,12 @@
+Invalid value
+
+sqrt :: Number -> Number
+        ^^^^^^
+          1
+
+At position 1:
+  Where: 1st Argument
+  Value: null
+  Types: Null
+
+Expected types: Number.
```

Comment:
All error messages updated to use this format. I removed the `.only` from the describe block I added, so you can see a bunch more error messages if you run the test suite.